### PR TITLE
Phol/driver document update [DRV-239]

### PR DESCRIPTION
**Pull request by PHOL-DA**

### CODE REVIEW

File: docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst [hunk 1]
```diff
@@ -26,16 +26,17 @@ Prerequisites
 ^^^^^^^^^^^^^
 
 To run the demo, you will need access
-to a Daml Enterprise release, the
+to a Daml Enterprise with drivers release, the
```
Comment (mziolekda) [inline docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst:29]: Unless the "Daml Enterprise with drivers" is a defined product line I would say "Daml Enterprise release with drivers"

File: docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst [hunk 2]
```diff
@@ -26,16 +26,17 @@ Prerequisites
...
 (`instructions here <https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Installation-Options/Install-Binaries>`_)
 installed.
 
+
 Introduction
 ^^^^^^^^^^^^
 
 The demo Ethereum deployment can be found inside the examples
-directory of the Canton Enterprise release. Unpack the Canton Enterprise release
+directory of the Canton Drivers release. Unpack the Canton Drivers release
```
Comment (mziolekda) [inline docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst:39]: Is the "Canton Drivers" mentioned here the same release artifact that was called "Daml Enterprise with drivers release" few lines up? If yes, I would consider consolidating the nomenclature.

File: docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst [hunk 3]
```diff
@@ -251,9 +252,9 @@ Manual deployment
 ^^^^^^^^^^^^^^^^^
 
 If you want to manually deploy ``Sequencer.sol`` to your Ethereum network, the file
-``<<canton-release/examples/e03-ethereum-sequencer/qbft-testnet/Sequencer-binary``
-contains the compiled Solidity code you need to deploy. For Besu, for example, you will need to specify the
-contents of ``Sequencer-binary`` in ``"code": "..."`` as documented `here <https://besu.hyperledger.org/en/stable/HowTo/Configure/Contracts-in-Genesis/#pre-deploying-contracts-in-the-genesis-file>`_.
+``<<canton-drivers-release>>/examples/e03-ethereum-sequencer/qbft-testnet/Sequencer.bin-runtime``
+contains the compiled Solidity code you need to deploy. For Besu, for example, you will need to specify the contents of ``Sequencer.bin-runtime`` in ``"code": "..."`` as documented `here <https://besu.hyperledger.org/stable/private-networks/how-to/configure/contracts>`_.
+This can also be seen in the ``generate-testnet.sh``. When code is added to the genesis block, the constructor is not used to initiate, so we need to do values instanced with ``"storage": "..."``.
```
Comment (mziolekda) [inline docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst:257]: This sentence is not clear to me. I trip over "is not used to initiate" and "we need to do values"

Comment (loiskilberg-da) [reply docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst:257]: Agree that the last sentence could be clearer. I don't know enough to edit it.

File: docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst (hunk 1 above)
Comment (PHOL-DA) [reply docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst]: ammended

File: docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst (hunk 2 above)
Comment (PHOL-DA) [reply docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst]: amended

File: docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst (hunk 3 above)
Comment (PHOL-DA) [reply docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst]: does the new wording sound better to you? tried to update it a bit

File: docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst [hunk 4]
```diff
@@ -251,9 +252,23 @@ Manual deployment
...
 
 If you want to manually deploy ``Sequencer.sol`` to your Ethereum network, the file
-``<<canton-release/examples/e03-ethereum-sequencer/qbft-testnet/Sequencer-binary``
-contains the compiled Solidity code you need to deploy. For Besu, for example, you will need to specify the
-contents of ``Sequencer-binary`` in ``"code": "..."`` as documented `here <https://besu.hyperledger.org/en/stable/HowTo/Configure/Contracts-in-Genesis/#pre-deploying-contracts-in-the-genesis-file>`_.
+``<<canton-drivers-release>>/examples/e03-ethereum-sequencer/qbft-testnet/Sequencer.bin-runtime``
+contains the compiled Solidity code you need to deploy. For Besu, for example, you will need to specify the contents of ``Sequencer.bin-runtime`` in ``"code": "..."`` as documented `here <https://besu.hyperledger.org/stable/private-networks/how-to/configure/contracts>`_.
+This can also be seen in the ``generate-testnet.sh``.
+
+When a contract is deployed using the ``genesis.json`` the constructor is never called. Therefore any variables initialized as part of the construct needs to be set using the ``"storage": "..."``.
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst:259]:
```suggestion
When a contract is deployed using the ``genesis.json``, the constructor is never called. Therefore, any variables initialized as part of the construct need to be set using the ``"storage": "..."``.
```

**Review by loiskilberg-da — APPROVED** [review]
Looks good! Just made a couple very minor suggestions to the sentence.

File: docs/2.7.0/docs/canton/usermanual/domains/fabric.rst [hunk 5]
```diff
@@ -10,14 +10,13 @@ Fabric Domain
...
 
-
 Tutorial
 --------
 
 To run the demo Canton Fabric deployment, you will need access to the following:
 
 
-- a Daml Enterprise release for access to the example files and the Canton binary
+- a Daml Enterprise with drivers release for access to the example files and the Canton binary
```
Comment (mziolekda) [inline docs/2.7.0/docs/canton/usermanual/domains/fabric.rst:19]:
```suggestion
- a Daml Enterprise release with drivers for access to the example files and the Canton binary
```
Noticed only now, that we have another instance

File: docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst (hunk 3 above)
Comment (mziolekda) [reply docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst]: Much clearer, thank you!

**Review by mziolekda — APPROVED** [review]

**Pull request merged**